src/features/navbar/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,107 @@
+'use client';
+
+import './Navbar.scss';
+
+import { AppStrings as AppT } from '@core/dictionaries';
+import { Routes } from '@core/values';
+import { NavbarSize } from '@features/navbar';
+import { useNavbarSize } from '@features/navbar/hooks';
+import { NavbarConfig } from '@features/navbar/values/navbar-config';
+import { useAuthStore } from '@processes/auth';
+import { FontIcon } from '@shared/ui';
+import clsx from 'clsx';
+import { useRouter } from 'next/navigation';
+import { useState } from 'react';
+
+import NavbarLogo from '../NavbarLogo/NavbarLogo';
+import NavbarSizeControl from '../NavbarSizeControl/NavbarSizeControl';
+import NavbarVisibilityControl from '../NavbarVisibilityControl/NavbarVisibilityControl';
+
+const CATEGORIES = NavbarConfig.categories;
+const CATEGORIES_COUNT = CATEGORIES.length - 1;
+
+interface Props {
+  defaultSize?: NavbarSize;
+}
+
+function Navbar({ defaultSize }: Props) {
+  const router = useRouter();
+  const { navbarSize, setNavbarSize } = useNavbarSize(defaultSize);
+
+  const authorized = useAuthStore((state) => state.authorized);
+  const [isOpen, setVisibility] = useState(false);
```

Comment:
For useState it's common to follow the naming pattern:
```typescript
const [state, setState] = useState(false);
```

So in this case the function should be renamed to `setIsOpen`